dbms/src/Flash/executeQuery.cpp

```diff
@@ -163,7 +163,7 @@ QueryExecutorPtr executeAsBlockIO(Context & context, bool internal)
 QueryExecutorPtr queryExecute(Context & context, bool internal)
 {
     if (context.getSettingsRef().enable_planner
-        && context.getSettingsRef().enable_pipeline
+        && (context.getSettingsRef().enable_pipeline || context.getSettingsRef().force_enable_pipeline)
```

Comment:
So `force_enable_pipeline` still does not take affect if `enable_planner == false`?

Reply:
My mistake, fixed.
When `force_enable_pipeline` is true, the pipeline model will be used regardless of whether the planner is turned on.